dist/docker/redhat/commandlineparser.py

```diff
@@ -4,7 +4,7 @@
 def parse():
     parser = argparse.ArgumentParser()
     parser.add_argument('--developer-mode', default='1', choices=['0', '1'], dest='developerMode')
```

Comment:
Should be folded with previous patch, to avoid bisect breakage (unless we manage to make experimental: true work)